src/store/modules/accounts/hdWallet.js

```diff
@@ -44,30 +49,30 @@ export default {
       state.nextAccountIdx += 1;
     },
     signWithoutConfirmation({ rootGetters: { account } }, data) {
-      return Crypto.sign(data, account.secretKey);
+      return sign(data, account.secretKey);
     },
     async confirmRawDataSigning({ dispatch }, data) {
       await dispatch('modals/open', { name: MODAL_CONFIRM_RAW_SIGN, data }, { root: true });
     },
-    async confirmTxSigning({ dispatch }, { encodedTx, host }) {
+    async confirmTxSigning({ dispatch }, { txBase64, host }) {
       let txObject;
       try {
-        txObject = TxBuilder.unpackTx(encodedTx, true).tx;
+        txObject = unpackTx(txBase64).tx;
```

Comment:
```suggestion
        txObject = unpackTx(txBase64);
```

Reply:
Resolving as this will be part of a separate PR